news/2024/2024-07-29-the-followpoint-flaro-the-queen-of-speed.md

```diff
@@ -0,0 +1,231 @@
+---
+layout: post
+title: "The Followpoint: Flaro, the Queen of Speed"
+date: 2024-07-29 17:00:00 +0000
+series: The Followpoint
+---
+
+Wake up babe, a new interview with [Flaro](https://osu.ppy.sh/users/10323184) just dropped.
```

Comment:
i disagree with putting a meme as the preview

Reply:
Do you have any suggestions on what this should be?

Reply:
Adding onto this, I don't see an issue why it shouldn't be a meme. I see a couple other newsposts have made their previews as memes.

Reply:
if it fits the tone we generally go for. they also usually give an idea of what's in the article

```suggestion
A new top player interview just dropped. This time: speed meta opinions and the cosplay buff.
```

Reply:
Fair enough I guess